js/background.js

```diff
@@ -50,9 +50,11 @@ function Background() {
             epoch = isDST ? estEpoch - oneHour : estEpoch,
             timeSinceEpoch = new Date().getTime() - epoch,
             majorVersion = Math.ceil(timeSinceEpoch / oneWeek),
-            minorVersion = Math.ceil(timeSinceEpoch % oneWeek / oneDay);
+            minorVersion = Math.ceil(timeSinceEpoch % oneWeek / oneDay), 
+            atb = 'v' + majorVersion + '-' + minorVersion;
 
-        localStorage['atb'] = 'v' + majorVersion + '-' + minorVersion;
+        localStorage['atb'] = atb;
+        localStorage['set_atb'] = atb;
```

Comment:
So is there any chance of it getting to onComplete and not being set?  Should we be checking set_atb there instead of just atb?